specification/tables/invoices/columns/invoiceid.md

```diff
@@ -0,0 +1,42 @@
+# Invoice ID
+
+An Invoice ID is a provider-assigned identifier for an invoice encapsulating some or all [*charges*](#glossary:charge) in the corresponding [*billing period*](#glossary:billing-period) for a given [*billing account*](#glossary:billing-account). Invoices are commonly used for scenarios like tracking billing transactions, facilitating payment processes and for performing invoice reconciliation between *charges* and billing periods.
+
+The InvoiceId column adheres to the following requirements:
+
+* InvoiceId is RECOMMENDED to be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
```

Comment:
I would expect to see a table of columns that should be in each dataset with their required level of inclusion. InvoiceId may be RECOMMENDED in cost dataset, but would be a MUST in the invoice dataset.